content/blogposts/2023/announcing-the-sourcegraph-app.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Announcing the Sourcegraph app"
+publishDate: 2023-03-23T10:00-07:00
+description: The Sourcegraph app, now in beta, brings code intelligence to your local machine in a lightweight, single-binary package.
+tags: [blog, release]
+slug: "announcing-sourcegraph-app"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/announcing-the-sourcegraph-app.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/announcing-the-sourcegraph-app.png
+---
+
+We’ve all felt it: navigating Big Code can be _really_ hard. To get the answers you need in a complex codebase, you might end up cloning a bunch of repos onto your local machine, or wasting time toggling between your code host and your IDE, or even writing your own hacky scripts to sync remote code locally.
+
+Sourcegraph brings everything you need to know about your code into a single, unified interface, and now it’s easier than ever to get it up and running on your local machine because today we’re launching the Sourcegraph app!
+
+It’s a free, lightweight single-binary that lets you run powerful code search across all the code you care about (both local and remote) right alongside your IDE.
```

Comment:
"The Sourcegraph app is a free . . . " (instead of "it's") reads a little better to my ear, but I defer to you.

Reply:
```suggestion
The Sourcegraph app is a free, lightweight single-binary that lets you run powerful code search across all the code you care about (both local and remote) right alongside your IDE.
```